<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="romanceButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Romance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voidButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(54, 52, 59); /* Light gray background */
    border-radius: 10px;
color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Chose a Genre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dramaButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Drama</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horrorButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Horror</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sportsButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Sports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comedyButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{; /* Light gray background */
	background-color: rgb(35, 35, 35);
    border-radius: 10px;
color: rgb(255, 255, 255);}
QPushButton:hover{background-color: rgb(50,50,50); /* Light gray background */
    border-radius: 10px;
	color: rgb(255, 255, 255);
 border: 2px solid #ccc;}</string>
   </property>
   <property name="text">
    <string>Comedy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
